src/components/Editor/EditorMenu.js

```diff
@@ -96,7 +103,7 @@ function getMenuItems(
     id: "node-menu-blackbox",
     label: toggleBlackBoxLabel,
     accesskey: blackboxKey,
-    disabled: false,
+    disabled: isGenerated,
```

Comment:
so... i think the comment before might help...
we want it to be disabled for pretty printed files, but nothing else...